root/components/UserAccountLayout.js

```diff
@@ -66,7 +66,7 @@ const UserAccountLayout = ({
     {...layoutProps}
   >
     <h1>
-      <EditorLink avatarSize={54} editor={user} />
+      <EditorLink avatarSize={31} editor={user} />
```

Comment:
FWIW our CSS for other headers says `background-size: 32px`

Reply:
Thanks, changed!